content-fragment-export-to-target/src/aem-cf-console-admin-1/ext.config.yaml

```
@@ -29,3 +29,13 @@ runtimeManifest:
           annotations:
             require-adobe-auth: false
             final: true
+        publish:
+          function: actions/publish/index.js
+          web: 'yes'
+          runtime: nodejs:16
```

```suggestion
          runtime: nodejs:18
```
See https://developer.adobe.com/runtime/docs/guides/reference/runtimes/
```
Node v18 is the default image and the one used to create pre-warm containers.
```

---

updated